src/main/java/lib/Injector.java

```diff
@@ -0,0 +1,41 @@
+package lib;
+
+import controller.ConsoleHandler;
+import dao.BetDao;
+import dao.BetDaoImpl;
+import dao.HumanDao;
+import dao.HumanDaoImpl;
+import factory.BetDaoFactory;
+import factory.HumanDaoFactory;
+
+import java.lang.reflect.Field;
+
+public class Injector {
+
+    public static void injectDependency() throws IllegalAccessException {
+        Class<ConsoleHandler> consoleHandlerClass = ConsoleHandler.class;
+        Class<BetDaoImpl> betDaoImplClass = BetDaoImpl.class;
+        Class<HumanDaoImpl> humanDaoImplClass = HumanDaoImpl.class;
+
+        Field[] consoleHandlerfields = consoleHandlerClass.getDeclaredFields();
+        for (Field field : consoleHandlerfields) {
+            if (field.getDeclaredAnnotation(Inject.class) != null) {
+                field.setAccessible(true);
```

Comment:
move this field before `field.set(null, BetDaoFactory.getBetDao());` in each `if` statement